web/server/codechecker_server/api/product_server.py

```diff
@@ -246,7 +246,9 @@ def getProductConfiguration(self, product_id):
         with DBSession(self.__session) as session:
             product = session.query(Product).get(product_id)
             if product is None:
-                LOG.error("Product with ID %d does not exist!", product_id)
+                msg = "Product with ID {0} does not exist!".format(product_id)
```

Comment:
This type of text format can not be optimized by the python log library. What was the pylint error here?

Reply:
The `msg` variable was undefined below. We use it in multiple places, so this is the reason why I moved it to a separate variable. We use this technique in multiple places in this file.